servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Completable.java

```diff
@@ -1187,6 +1187,24 @@ public static Completable completed() {
         return CompletedCompletable.INSTANCE;
     }
 
+    /**
+     * <p>
```

Comment:
We generally just use a `<p/>` to indicate a new paragraph, so you do not have to wrap the paragraph between `<p>...</p>`